src/solidlsp/language_servers/clojure_lsp/initialize_params.json

```diff
@@ -0,0 +1,133 @@
+{
```

Comment:
I think there's no point for keeping these jsons separately, for the pyright server I put everything directly into the module. Wdyt? If you agree, would you mind moving them to dicts and then deleting the jsons?

Reply:
JSONs make no sense, agreed.